python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -85,6 +86,8 @@ class ImageDifferenceConfig(pexConfig.Config):
     doWriteSubtractedExp = pexConfig.Field(dtype=bool, default=True, doc="Write difference exposure?")
     doWriteMatchedExp = pexConfig.Field(dtype=bool, default=False,
                                         doc="Write warped and PSF-matched template coadd exposure?")
+    doWriteWarpedTemplate = pexConfig.Field(dtype=bool, default=False,
```

Comment:
If this config only affects/applies-to/works-with the ZogyImagePsfMatchTask subtask, It should be in the configuration of ZogyImagePsfMatchTask.